org.bridgedb.bio/resources/org/bridgedb/bio/datasources_headers.txt

```diff
@@ -2,17 +2,17 @@
 #
 # The term "conventional", as found in the descriptions below, refers to established usage norms for the BridgeDb project.
 #
-# Tab-delimited table:
+# Tab-delimited table:				
 #
-# column	header	description	example_entry
-1	datasource_name	Conventional name for datasource	Affy
-2	system_code	Conventional 1-3 letter code for datasource	X
-3	website_url	URL to main webpage for datasource	http://www.affymetrix.com/
-4	linkout_pattern	URL pattern for constructing links to ID-specific datasource pages, using $id as substitution variable	https://www.affymetrix.com/LinkServlet?probeset=$id
-5	example_identifier	A valid example of a datasource identifier; may not be representative of all types of identifiers from a given resource	1851_s_at
-6	entity_identified	Type of entity (e.g., biological molecule or concept) identified by datasource	probe
-7	single_species	Specific species represented by datasource (latin genus species); blank if datasource represents more than one species	Homo sapiens
-8	identifier_type	Conventional usage type for datasource: 1 = primary identifiers (used to uniquely identify molecules or concepts; useful for identifying entities on a pathway; useful targets for mapping measurement data), 0 = secondary identifiers (used to annotate molecules or concepts; useful as secondary linkouts; useful sources for data/annotation mapping)	0
-9	uri	Official URI for datasource (e.g., from miriam); or repeat of system_code if unknown or unregistered	urn:miriam:affy.probeset
-10	regex	Regular expression capturing all possible identifiers as narrowly as possible for datasource (see miriam)	\d{4,}((_[asx])?_at)?
-11	official_name	Official name of datasource resource; useful as an end-user display name; useful in name resolution	Affymetrix Probeset
+# column	header	description	example_entry	@id
+1	datasource_name	Conventional name for datasource	Affy	http://vocabularies.bridgedb.org/ops#conventionalName
+2	system_code	Conventional 1-3 letter code for datasource	X	http://vocabularies.bridgedb.org/ops#systemCode
+3	website_url	URL to main webpage for datasource	http://www.affymetrix.com/	http://vocabularies.bridgedb.org/ops#mainUrl
+4	linkout_pattern	"URL pattern for constructing links to ID-specific datasource pages, using $id as substitution variable"	https://www.affymetrix.com/LinkServlet?probeset=$id	http://vocabularies.bridgedb.org/ops#UriPattern
```

Comment:
Is "linkout_pattern" a property and "http://vocabularies.bridgedb.org/ops#UriPattern" a class?